portal.php

```diff
@@ -480,17 +482,19 @@ function hasScrolled() {
         }
     </script>
 
-    <script>
-        function on() {
-            document.getElementById("myModal").style.display = "block";
-        }
-
-        function off() {
-            document.getElementById("myModal").style.display = "none";
-        }
-    </script>
-    
     <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js" type="text/javascript"></script>
 
+    <script>
+    window.onload = function(){

```

Comment:
instead of this use $_SESSION['msg'] already written on line 189 to show your messages after submitting the form

Reply:
Done